odbcconfig/odbcinst.ini

```diff
@@ -4,4 +4,4 @@
 
 [EXAODBC]
 #Driver location will be appended in build environment:
-#DRIVER=<build_dir>/driver/libexaodbc-uo2214.so
+DRIVER=/home/nic/Projects/sqlalchemy-exasol/driver/libexaodbc-uo2214lv1.so
```

Comment:
you are leaking local paths

Reply:
good catch! I think this file isn't needed anymore -> I will remove it